config/initializers/statsd.rb

```diff
@@ -0,0 +1,28 @@
+Notifier.extend StatsD::Instrument
+Notifier.statsd_count_success :perform, 'Webhook.perform.success'
+
+Indexer.extend StatsD::Instrument
+Indexer.statsd_count_success :perform, 'Indexer.perform.success'
+Indexer.statsd_measure :perform, 'Indexer.perform'
+
+SearchesController.extend StatsD::Instrument
+SearchesController.statsd_measure :show, 'SearchesController.show'
```

Comment:
Do you mean add tests for them or just make sure they work? Also just realized I don't need the explicit search one anymore because of the generic one down below.